packages/semcom-demo-app/lib/demo.component.ts

```diff
@@ -25,16 +25,16 @@ export class DemoComponent extends RxLitElement {
   private actor = interpret(this.machine, { devTools: true }).onTransition((appState) => console.log(appState.value));
 
   @state()
-  state: State<DemoContext>;
+  state!: State<DemoContext>;
 
   @state()
-  tags: string[];
+  tags!: string[];
 
   @state()
-  session: Session;
+  session!: Session;
 
   @query('.components')
-  contentElement: HTMLDivElement;
+  contentElement!: HTMLDivElement;
```

Comment:
Hmmmm, do we really want to do this? It circumvents typechecking, just so we would not have to check for undefined...

Reply:
Moreover, RxLitElement's subscribe function is not failsafe: if there's nothing in the context, the variable will in fact stay undefined, I think.